### ISSUE TITLE
Product Attributes Just Spins 

### ISSUE THREAD

**Issue by krushendo** [woocommerce/woocommerce#15110]
### EXPLANATION OF THE ISSUE

Everything is 100% up to date including Woo to 3.0.7 and WP to 4.7.4, all plugins are updated, my theme Porto is fully up to date and I have removed the dependent WooCommerce files so it will check the plugin instead of the theme.

### STEPS TO REPRODUCE THE ISSUE

Any variable product, try to add attributes (I need to add sizes... nothing happens.)

### SYSTEM STATUS

<details>
```
`
### WordPress Environment ###

Home URL: http://lifebloomcreative.com
Site URL: http://lifebloomcreative.com
WC Version: 3.0.7
Log Directory Writable: ✔
WP Version: 4.7.4
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US

### Server Environment ###

Server Info: Apache
PHP Version: 5.6.27
PHP Post Max Size: 65 MB
PHP Time Limit: 30
PHP Max Input Vars: 1000
cURL Version: 7.45.0
OpenSSL/1.0.1e

SUHOSIN Installed: –
MySQL Version: 5.6.32
Max Upload Size: 64 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 3.0.7
WC Database Prefix: wp_vc17gpsag3_
woocommerce_sessions: ✔
woocommerce_api_keys: ✔
woocommerce_attribute_taxonomies: ✔
woocommerce_downloadable_product_permissions: ✔
woocommerce_order_items: ✔
woocommerce_order_itemmeta: ✔
woocommerce_tax_rates: ✔
woocommerce_tax_rate_locations: ✔
woocommerce_shipping_zones: ✔
woocommerce_shipping_zone_locations: ✔
woocommerce_shipping_zone_methods: ✔
woocommerce_payment_tokens: ✔
woocommerce_payment_tokenmeta: ✔

### Security ###

Secure connection (HTTPS): ❌Your store is not using HTTPS. Learn more about HTTPS and SSL Certificates.
Hide errors from visitors: ❌Error messages should not be shown to visitors.

### Active Plugins (24) ###

ManageWP - Worker: by ManageWP – 4.2.18
Akismet Anti-Spam: by Automattic – 3.3.2
Better WordPress reCAPTCHA: by Khang Minh – 2.0.3
Contact Form 7 Datepicker: by Aurel Canciu – 2.6.0
Contact Form 7: by Takayuki Miyoshi – 4.7
Duplicate Post: by Enrico Battocchi – 3.2
Envato WordPress Toolkit: by Envato – 1.7.3
Jkreativ Mail Subscribe List: by Richard Leishman t/a Webforward modified by Jegtheme – 1.0.0
Jkreativ Plugin: by Agung Bayu Iswara – 1.2.1
WPBakery Visual Composer: by Michael M - WPBakery.com – 5.1.1
Lazy Load: by  – 0.6.1
Lightbox Plus Colorbox: by Dan Zappone – 2.7.2
Pantheon Advanced Page Cache: by Pantheon – 0.1.4
Really Simple CAPTCHA: by Takayuki Miyoshi – 1.9
Remove Query Strings From Static Resources: by Speed Up My Website
Your WP Expert – 1.4

Simple Page Ordering: by Jake Goldman
10up – 2.2.4

UpdraftPlus - Backup/Restore: by UpdraftPlus.Com
DavidAnderson – 1.13.1

WooCommerce Square: by WooCommerce – 1.0.16 – 1.0.17 is available
WooCommerce: by Automattic – 3.0.7
WordPress Importer: by wordpressdotorg – 0.6.3
Yoast SEO: by Team Yoast – 4.7.1
WP-Optimize: by David Anderson
Ruhani Rabin
Team Updraft – 2.1.1

WP Retina 2x: by Jordy Meow – 5.0.5
WP Smush: by WPMU DEV – 2.6.3

### Settings ###

API Enabled: ✔
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)


### WC Pages ###

Shop base: #2627 - /shop/
Cart: #2628 - /cart/
Checkout: #2629 - /checkout/
My account: #2630 - /my-account/

### Theme ###

Name: Lifebloom Creative
Version: 2.3.2
Author URL: http://jegtheme.com/
Child Theme: ✔
Parent Theme Name: Jkreativ
Parent Theme Version: 2.4.2
Parent Theme Author URL: http://jegtheme.com/
WooCommerce Support: ✔

### Templates ###

Archive Template: Your theme has a woocommerce.php file
you will not be able to override the woocommerce/archive-product.php custom template since woocommerce.php has priority over archive-product.php. This is intended to prevent display issues.

Overrides: –
`
```
</details>

**Comment by squiggle78** [comment]
I have the same problem and needed to go back to 2.6.14 before the problem was solved, but on that release, my variant prices are all a mess - some with taxes and others without :(

**Comment by mikejolley** [comment]
This will be a conflict with another plugin - see my answer here https://github.com/woocommerce/woocommerce/issues/14972#issuecomment-300402820

**Comment by krushendo** [comment]
I've deactivated every plugin besides the themes required plugin, I'm having issues with Woo writing the debug log because of permissions which may also be the issue.

And now it looks like its conflicting with the themes main plugin.... great. I ticketed them like last week and no response... guess I will retheme the whole site....

Error: Option 'ajax' is not allowed for Select2 when attached to a element. at String. (select2.min.js:41) at Function.each (load-scripts.php:2) at d.prepareOpts (select2.min.js:41) at d.prepareOpts (select2.min.js:41) at d.init (select2.min.js:41) at HTMLSelectElement. (select2.min.js:43) at Function.each (load-scripts.php:2) at a.fn.init.each (load-scripts.php:2) at a.fn.init.a.fn.select2 (select2.min.js:43) at HTMLSelectElement. (wc-enhanced-select.min.js:1) select2.min.js:41 Uncaught Error: Option 'ajax' is not allowed for Select2 when attached to a element.
at String. (select2.min.js:41)
at Function.each (load-scripts.php:2)
at d.prepareOpts (select2.min.js:41)
at d.prepareOpts (select2.min.js:41)
at d.init (select2.min.js:41)
at HTMLSelectElement. (select2.min.js:43)
at Function.each (load-scripts.php:2)
at a.fn.init.each (load-scripts.php:2)
at a.fn.init.a.fn.select2 (select2.min.js:43)
at HTMLSelectElement. (wc-enhanced-select.min.js:1)
